protobuf/concept.proto

```diff
@@ -105,9 +105,8 @@ message Concept {
 message Thing {
 
     bytes iid = 1;
-    Encoding encoding = 2;
-    AttributeType.ValueType value_type = 3;
-    Attribute.Value value = 4;
```

Comment:
`value_type` is now redundant (equivalent to `type.value_type`) and `encoding` can be inferred from `type.encoding`.